src/ucp/rma/flush.c

```diff
@@ -633,6 +633,10 @@ ucp_worker_flush_nbx_internal(ucp_worker_h worker,
     req->flush_worker.uct_flags  = uct_flags;
     req->flush_worker.prog_id    = UCS_CALLBACKQ_ID_NULL;
 
+    ucs_list_print_links(&worker->all_eps, worker->num_all_eps);
```

Comment:
Please remove prints before and after the shuffle.

Reply:
done